app/admin/applicants/page.tsx

```
@@ -0,0 +1,532 @@
+import { Avatar, AvatarFallback, AvatarImage } from "@/components/ui/basic/avatar"
+import { Badge } from "@/components/ui/basic/badge"
+import { Search, Filter, Download, Users, Mail, User, Clock, FileText, Upload, UserPlus, FileSpreadsheet } from "lucide-react"
+import { Tabs, TabsContent, TabsList, TabsTrigger } from "@/components/ui/navigation/tabs"
+import { extendedPalette } from "@/lib/colors"
+import { Skeleton } from "@/components/ui/feedback/skeleton"
+import { cn } from "@/lib/utils"
+import { MultiPurposeModal } from "@/components/ui/overlay/multi-purpose-modal"
+import { Label } from "@/components/ui/basic/label"
+
+// Define User interface to match actual structure
+interface User {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'User'. Consider to delete it or rename it.

'User' is defined here:


(lint/suspicious/noRedeclare)